tests/sentry/integrations/utils/test_assert_metrics.py

```diff
@@ -0,0 +1,19 @@
+from sentry.integrations.types import EventLifecycleOutcome
```

Comment:
this utility file is also in the diff for https://github.com/getsentry/sentry/pull/80530, will get dropped from this pr's diff if the other pr is merged first. i just have it here because some tests depend on it.